src/auth/ejabberd_auth_riak.erl

```diff
@@ -65,7 +65,10 @@ set_password(LUser, LServer, Password) ->
             {error, invalid_password};
         Password ->
             User = mongoose_riak:fetch_type(bucket_type(LServer), LUser),
-            do_set_password(User, LUser, LServer, Password)
+            do_set_password(User, LUser, LServer, Password);
+        {<<>>, Scram} ->
```

Comment:
Good idea and I agree this could be a part of separate PR. Having this covered in tests will expose once we break something for plaintext password format which from now on might not be visible (similarly to what was before with scram).